creator-node/src/services/sync/syncQueueJobProcessor.js

```diff
@@ -0,0 +1,40 @@
+const processSync = require('./processSync')
+const { logger } = require('../../logging')
+// Direct serviceRegistry require is necessary here since this module is run in a background process
+const { serviceRegistry } = require('../../serviceRegistry')
+
+/**
+ * Processing logic for all jobs picked off bull queue in `SyncQueueService`
+ *
+ * - This may be run in a background process as a sandboxed processor, as per configs in `SyncQueueService`
+ * - Swallows any sync failures silently - primary will handle re-triggering sync requests
+ * - @dev TODO - consider recording failures in redis
+ *
+ * @param {Object} job bull job info
```

Comment:
combine this file and syncQueueService.js

Reply:
so this was already done, that file was not used, see latest commits for diff